examples/tests/valid/parameters.test.w

```diff
@@ -0,0 +1,23 @@
+let app = nodeof(this).app;
+
+let registrar = app.parameterRegistrar;
+
+registrar.addParameterSchema({
+  type: "object",
+  properties: {
+    foo: {
+      type: "string"
+    },
+    meaning_of_life: {
+      type: "number"
+    }
+  }
+});
+
+let foo = registrar.getParameterValue("foo");
```

Comment:
I need to add a test of getting a value that is available, just need to tweak how the test would be generated in hangar